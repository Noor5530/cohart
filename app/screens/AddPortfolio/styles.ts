import { appFonts } from 'components/text';
import { iconColor, placeHolderColor, shimmerColor } from 'config/colors';
import { Platform, StyleSheet } from 'react-native';

import { textColor } from '../../config/colors';

const styles = StyleSheet.create({
  header: {
    height: 28.22,
    width: '100%',
    paddingLeft: 22.4,
    paddingRight: 22.4,
    alignItems: 'center',
    justifyContent: 'space-between',
    flexDirection: 'row',
    marginTop: 20,
  },
  descriptionView: {
    borderWidth: 1,
    borderColor: iconColor,
    borderRadius: 9,
    paddingLeft: 6,
    paddingRight: 6,
    paddingHorizontal: 17,
    height: 32,
    alignItems: 'center',
    justifyContent: 'center',
  },
  heading: {
    fontFamily: appFonts.InterRegular,
    fontStyle: 'normal',
    fontWeight: '500',
    fontSize: 22.0843,
    alignItems: 'center',
    textAlign: 'center',
    color: iconColor,
  },
  titleHeading: {
    fontFamily: appFonts.InterRegular,
    fontStyle: 'normal',
    fontWeight: '500',
    marginLeft: 22.4,
    paddingRight: 22.4,
    fontSize: 35,
    width: '85%',
    height: 110,
    color: iconColor,
  },
  input: {
    fontFamily: appFonts.InterRegular,
    fontStyle: 'normal',
    fontWeight: '500',
    marginLeft: 22.4,
    paddingRight: 22.4,
    fontSize: 35,
    width: '100%',
    height: 110,
    color: iconColor,
  },
  descriptionText: {
    fontFamily: appFonts.InterRegular,
    fontStyle: 'normal',
    fontWeight: 'normal',
    fontSize: 14,
    color: placeHolderColor,
    paddingLeft: 4,
  },
  footerView: {
    height: 350,
    width: '100%',
  },
  line: {
    width: 33.54,
    height: 3.59,
    alignSelf: 'center',
    backgroundColor: textColor,
    borderRadius: 5.98701,
  },
  footerHeader: {
    flexDirection: 'row',
    justifyContent: 'space-between',
    alignItems: 'center',
    width: '100%',
  },
  footerContainer: {
    width: '100%',
    height: '100%',
    paddingHorizontal: 33,
    paddingTop: 20,
  },
  userView: {
    flexDirection: 'row',
    alignItems: 'center',
  },
  userAvatar: {
    width: 32.05,
    height: 32.05,
    borderRadius: 24.9131,
    backgroundColor: shimmerColor,
  },
  userName: {
    fontFamily: appFonts.InterRegular,
    fontStyle: 'normal',
    fontWeight: '500',
    fontSize: 14.0529,
    paddingLeft: 16.4,
  },
  footerText: {
    fontFamily: appFonts.InterRegular,
    fontStyle: 'normal',
    fontWeight: 'bold',
    fontSize: 11,
  },
  footerTextInput: {
    paddingTop: 0,
    fontSize: 10,
    fontFamily: appFonts.InterRegular,
    fontStyle: 'normal',
    paddingLeft: 3,
    width: '60%',
    alignSelf: 'center',
    textAlignVertical: 'top',
  },
  descriptionTextInput: {
    fontSize: 10,
    fontFamily: appFonts.InterRegular,
    fontStyle: 'normal',
    paddingLeft: 3,
    width: '100%',
    height: 125,
    textAlign: 'left',
    paddingTop: 20,
    textAlignVertical: 'top',
  },
  tag: {
    paddingLeft: 8,
    paddingRight: 8,
    borderRadius: 6.49051,
    height: 25,
    alignItems: 'center',
    justifyContent: 'center',
    marginRight: 10,
    borderWidth: 1,
    borderColor: '#000000',
  },
  tagText: {
    fontFamily: appFonts.InterRegular,
    fontStyle: 'normal',
    fontWeight: 'normal',
    fontSize: 10,
  },
  searchBarContainer: {
    marginTop: 22,
    paddingLeft: 29,
    paddingRight: 29,
    height: 40,
    flexDirection: 'row',
  },
  searchBar: {
    borderRadius: 10,
    flex: 1,
    height: '100%',
    flexDirection: 'row',
    alignItems: 'center',
    paddingLeft: 10,
    paddingRight: 10,
  },
  searchInput: {
    fontFamily: appFonts.InterRegular,
    fontStyle: 'normal',
    fontWeight: '500',
    fontSize: 12,
    color: iconColor,
    height: 40,
    flex: 1,
  },
  bottomContainer: {
    flexDirection: 'row',
  },
  bottom: {
    flexDirection: 'row',
    width: '65%',
    height: 30,
  },
  switchContainer: {
    flexDirection: 'row',
    width: '45%',
  },
  switch: {
    transform: [{ scaleX: 0.5 }, { scaleY: 0.5 }],
    marginLeft: -10,
    marginTop: Platform.OS == 'ios' ? -8 : -10,
  },
  mediumHeading: { flexDirection: 'row', paddingTop: 5 },
  mediumView: { flexDirection: 'row', width: '65%' },
  priceView: { flexDirection: 'row', width: '45%' },
  descriptionTextView: {
    marginTop: 10,
    borderWidth: 0.5,
    borderColor: '#9D9D9D',
  },
  tagListView: { height: 30 },
});

export default styles;
