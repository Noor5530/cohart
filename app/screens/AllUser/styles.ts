import { StyleSheet } from 'react-native';

const styles = StyleSheet.create({
  constrainer: {
    height: '100%',
    backgroundColor: 'white',
  },
});
export default styles;
