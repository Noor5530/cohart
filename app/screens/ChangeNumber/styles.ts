import { appFonts } from 'components/text';
import { iconColor, textColor } from 'config/colors';
import { StyleSheet } from 'react-native';
import {
  heightPercentageToDP as hp,
  widthPercentageToDP as wp,
} from 'react-native-responsive-screen';

const style = StyleSheet.create({
  container: {
    height: hp(100),
    width: '100%',
    backgroundColor: textColor,
  },
  header: {
    flex: 1,
    width: '100%',
  },
  mainContainer: {
    flex: 1,
    width: '100%',
    paddingHorizontal: wp('5'),
  },
  bottomContainer: { flex: 1, width: '100%', alignItems: 'center' },
  button: {
    marginTop: hp(5),
    height: 35,
    width: 140,
    alignSelf: 'center',
    borderRadius: 30,
    borderWidth: 1,
    borderColor: iconColor,
    alignItems: 'center',
    justifyContent: 'center',
  },
  bottonText: {
    fontWeight: '500',
    fontFamily: appFonts.InterRegular,
    fontSize: 13,
  },
  bottomText: {
    flexDirection: 'row',
    alignItems: 'center',
    justifyContent: 'center',
    height: 40,
  },
  cross: {
    width: 20,
    height: 20,
    borderRadius: 10,
    borderWidth: 1,
    alignItems: 'center',
    justifyContent: 'center',
    marginRight: 10,
  },
  label: {
    fontSize: 13,
    color: iconColor,
  },
  reefersText: {
    paddingTop: 25,
    height: 55,
    textAlignVertical: 'top',
    fontSize: 13,
    color: iconColor,
  },
  textInput: {
    height: 50.42,
    padding: 0,
    margin: 0,
    width: '100%',
    color: iconColor,
    fontFamily: appFonts.InterRegular,
    fontSize: wp('5'),
    fontWeight: 'bold',
    backgroundColor: textColor,
    borderWidth: 0,
  },
  textContainer: {
    width: '100%',
    margin: 0,
    backgroundColor: textColor,
    height: 50.42,
  },
  countryContainerStyle: {
    borderWidth: 2,
    borderColor: iconColor,
    borderRadius: 10,
    paddingHorizontal: 10,
    backgroundColor: textColor,
  },
  codeTextStyle: {
    margin: 0,
    color: iconColor,
    fontFamily: appFonts.InterRegular,
    fontSize: wp('5'),
    fontWeight: 'bold',
  },
  inputContainerStyle: {
    padding: 0,
    margin: 0,
    width: '100%',
    alignItems: 'flex-start',
    justifyContent: 'flex-start',
    height: 50.42,
    borderColor: '#000000',
    borderRadius: 14.4067,
    backgroundColor: textColor,
  },
  heading: {
    paddingVertical: 20,
    fontFamily: appFonts.InterRegular,
    color: iconColor,
    fontWeight: '400',
    fontSize: 15,
  },
  changeNumberContainer: {
    flex: 1,
    width: '100%',
    justifyContent: 'center',
    paddingTop: hp(6),
  },
  numberInput: { flexDirection: 'row', height: 51 },
  changeNumberButton: { flex: 1 },
});
export default style;
