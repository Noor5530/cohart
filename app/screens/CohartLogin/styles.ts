import { appFonts } from 'components/text';
import { primary, textColor } from 'config/colors';
import { StyleSheet } from 'react-native';
import { heightPercentageToDP as hp, widthPercentageToDP as wp } from 'react-native-responsive-screen';

import { iconColor } from '../../config/colors';

const styles = StyleSheet.create({
  container: {
    paddingTop: hp(40),
    flex: 1,
    alignItems: 'center',
  },
  checkContainer: {
    flexDirection: 'row',
    width: '100%',
    alignItems: 'center',
    justifyContent: 'center',
    paddingTop: 23,
  },
  shortText: {
    width: 298.94,
    fontStyle: 'normal',
    fontWeight: 'normal',
    fontSize: 18.0084,
    lineHeight: 22,
    alignItems: 'flex-end',
    textAlign: 'center',
    color: textColor,
    fontFamily: appFonts.InterRegular,
  },
  checkBox: {
    width: 20,
    height: 20,
    borderRadius: 5.7147,
    alignItems: 'center',
    justifyContent: 'center',
    borderWidth: 0.476225,
    borderColor: iconColor,
    backgroundColor: textColor,
  },
  footer: {
    paddingLeft: 10,
    fontFamily: appFonts.InterRegular,
    fontStyle: 'normal',
    fontWeight: 'normal',
    fontSize: 14,
    textAlign: 'center',
    lineHeight: 17,
    color: textColor,
  },
  termsConditionLine: {
    paddingLeft: 3,
    fontFamily: appFonts.InterRegular,
    fontStyle: 'normal',
    fontWeight: 'normal',
    fontSize: 14,
    textAlign: 'center',
    lineHeight: 17,
    color: primary,
    textDecorationLine: 'underline',
    textDecorationColor: primary,
  },
  waitListButton: {
    fontFamily: appFonts.InterRegular,
    fontStyle: 'normal',
    fontWeight: '500',
    fontSize: 14,
    lineHeight: 17,
    alignItems: 'center',
    textAlign: 'center',
    color: primary,
    paddingTop: 10,
  },
  referral: {
    fontFamily: appFonts.InterRegular,
    fontStyle: 'normal',
    fontWeight: 'normal',
    fontSize: 15.6501,
    lineHeight: 23,
    textAlign: 'center',
    color: primary,
    height: 23,
    marginTop: hp('11%'),
    textTransform: 'uppercase',
  },
  bottomText: {
    fontFamily: appFonts.InterRegular,
    fontStyle: 'normal',
    fontWeight: '400',
    fontSize: 10.955,
    lineHeight: 16,

    textTransform: 'uppercase',
    position: 'absolute',
    bottom: hp(2),
    left: 41.24,
    color: textColor,
  },
  bottomInnerText: {
    fontFamily: appFonts.InterRegular,
    fontStyle: 'normal',
    fontWeight: '600',
    fontSize: 10.955,
    lineHeight: 16,

    textTransform: 'uppercase',
    color: textColor,
  },
  imageBackground: {
    height: hp(100),
    width: wp(100),
  },
});

export default styles;
