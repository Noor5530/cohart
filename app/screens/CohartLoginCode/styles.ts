import { appFonts } from 'components/text';
import { iconColor, primary, textColor } from 'config/colors';
import { StyleSheet } from 'react-native';
import { heightPercentageToDP as hp, widthPercentageToDP as wp } from 'react-native-responsive-screen';

const styles = StyleSheet.create({
  container: {
    justifyContent: 'flex-end',
    flex: 1,
    alignItems: 'center',
    paddingBottom: hp(20),
  },
  checkContainer: {
    flexDirection: 'row',
    width: '100%',
    alignItems: 'center',
    justifyContent: 'center',
    paddingTop: 23,
  },
  shortText: {
    width: 298.94,
    fontStyle: 'normal',
    fontWeight: 'normal',
    fontSize: 18.0084,
    lineHeight: 22,
    alignItems: 'flex-end',
    textAlign: 'center',
    color: textColor,
    fontFamily: appFonts.InterRegular,
  },
  checkBox: {
    width: 20,
    height: 20,
    borderRadius: 5.7147,
    alignItems: 'center',
    justifyContent: 'center',
    borderWidth: 0.476225,
    borderColor: iconColor,
    backgroundColor: textColor,
  },
  footer: {
    paddingLeft: 10,
    fontFamily: appFonts.InterRegular,
    fontStyle: 'normal',
    fontWeight: 'normal',
    fontSize: 14,
    textAlign: 'center',
    lineHeight: 17,
    color: textColor,
  },
  termsConditionLine: {
    paddingLeft: 3,
    fontFamily: appFonts.InterRegular,
    fontStyle: 'normal',
    fontWeight: 'normal',
    fontSize: 14,
    textAlign: 'center',
    lineHeight: 17,
    color: primary,
    textDecorationLine: 'underline',
    textDecorationColor: primary,
  },
  waitListButton: {
    fontFamily: appFonts.InterRegular,
    fontStyle: 'normal',
    fontWeight: '500',
    fontSize: 14,
    lineHeight: 17,
    alignItems: 'center',
    textAlign: 'center',
    color: primary,
    paddingTop: 10,
  },
  referral: {
    fontFamily: appFonts.InterRegular,
    fontStyle: 'normal',
    fontWeight: 'normal',
    fontSize: 15.6501,
    lineHeight: 23,
    textAlign: 'center',
    color: primary,
    height: 23,
    marginTop: hp('11%'),
    textTransform: 'uppercase',
  },
  bottomText: {
    fontFamily: appFonts.InterRegular,
    fontStyle: 'normal',
    fontWeight: 'normal',
    fontSize: 10.955,
    lineHeight: 16,
    textTransform: 'uppercase',
    position: 'absolute',
    bottom: 10,
    left: 41.24,
    color: textColor,
  },
  imageBackground: {
    height: hp(100),
    width: wp(100),
  },
  loginButton: {
    marginTop: hp('3'),
    height: 48,
    borderRadius: 30,
    paddingLeft: 15,
    paddingRight: 15,
    backgroundColor: primary,
    alignItems: 'center',
    justifyContent: 'center',
  },
  buttonTitle: {
    fontFamily: appFonts.InterRegular,
    fontStyle: 'normal',
    fontWeight: 'bold',
    fontSize: wp('3.8'),

    textAlign: 'center',
    color: iconColor,
  },
});

export default styles;
