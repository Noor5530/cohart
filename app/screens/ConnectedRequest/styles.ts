import { StyleSheet } from "react-native";
import { widthPercentageToDP, heightPercentageToDP } from "react-native-responsive-screen";
import { iconColor, neonYellow, textColor } from "config/colors";
import { appFonts } from "../../components/text";
const styles = StyleSheet.create({
  ProfileConatainer: {
    flex: 1, justifyContent: "space-between"

  },
  column: {
    paddingVertical: heightPercentageToDP(0),
    flexDirection: 'column',
    alignItems: 'center',
    height: heightPercentageToDP(65),
    paddingTop: heightPercentageToDP(5)
         //THIS LINE HAS CHANGED
  },
  paragraph: {
    width: widthPercentageToDP("80%"),
    flex: 1,
    flexDirection: 'column',
    alignItems: 'center',       //THIS LINE HAS CHANGED
    paddingTop: widthPercentageToDP("8%"),
  },
  fansHeader: {
    backgroundColor: neonYellow,
    paddingBottom: '3%'
  },
  container: {
    flex: 1, justifyContent: "space-between"
  },
  avatar: {
    height: widthPercentageToDP("28%"),
    width: widthPercentageToDP("28%"),
    borderWidth: 1,
    borderRadius: widthPercentageToDP("14%"),
    backgroundColor: 'lightgray'
  },
  contentStyle: {
    paddingBottom: 30,
  },
  avatarImage: {
    height: widthPercentageToDP("16%"),
    width: widthPercentageToDP("16%"),
    borderWidth: 2,
    borderRadius: widthPercentageToDP("8%"),
    //  paddingHorizontal: widthPercentageToDP("5%"),
  },

  button: {
    width: '105%',
    height: '105%',
    marginVertical: '65%',
    position: 'absolute',
    justifyContent: 'center',
    alignItems: 'center',
    alignSelf: 'center',

  },

  followingBackButton: {
    width: widthPercentageToDP(32),
    alignItems: "center",
    justifyContent: "center",
    paddingVertical: 5,
    borderWidth: 1,
    borderColor: iconColor,
    marginBottom: 20,
    backgroundColor: iconColor,
  },
  title: {
    paddingVertical: widthPercentageToDP("5%"),
    width: widthPercentageToDP("50"),
    fontWeight: '500',
    fontSize: 20,
    textAlign: 'center'

  },
  subTitle: {
    textAlign: 'center',
    paddingVertical: widthPercentageToDP("5%"),
    fontWeight: '400',
    fontSize: 19,
  },
  buttonText: {
    fontSize: widthPercentageToDP(4.5),
    color: textColor,
    fontFamily: appFonts.InterRegular,
    fontWeight: "500",
    textTransform: "uppercase",

  },

  followingButtonText: {
    fontSize: widthPercentageToDP(3.5),
    color: textColor,
    fontFamily: appFonts.InterRegular,
    fontWeight: "600",
    textTransform: "uppercase",
  },
  followingBackButtonText: {
    fontSize: widthPercentageToDP(3.5),
    color: textColor,
    fontFamily: appFonts.InterRegular,
    fontWeight: "600",
    textTransform: "uppercase",
  },
});
export default styles;
