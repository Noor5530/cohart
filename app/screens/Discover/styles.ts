import { StyleSheet } from 'react-native';
import { heightPercentageToDP as hp } from 'react-native-responsive-screen';

import { appFonts } from '../../components/text';
import { iconColor, primary } from '../../config/colors';

const styles = StyleSheet.create({
  text: {
    paddingTop: 5,
    textAlign: 'auto',
    color: iconColor,
    fontFamily: appFonts.InterBold,
    fontSize: 20,
    fontWeight: '500',
  },
  acceptButton: {
    position: 'absolute',
    height: hp('4'),
    width: '100%',
    borderRadius: 20,
    alignItems: 'center',
    justifyContent: 'center',
    backgroundColor: primary,
    flexDirection: 'row',
    bottom: 20,
    borderColor: iconColor,
  },
  footerButton: {
    height: 42,
    width: 168,
    borderRadius: 20,
    alignItems: 'center',
    justifyContent: 'center',
    backgroundColor: primary,
    flexDirection: 'row',
    marginBottom: 20,
  },
});
export default styles;
