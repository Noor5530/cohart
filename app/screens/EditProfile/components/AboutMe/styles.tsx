import { appFonts } from "components/text";
import { StyleSheet } from "react-native";
import { widthPercentageToDP } from "react-native-responsive-screen";

const styles = StyleSheet.create({
  fullName: {
    fontFamily: appFonts.InterSemiBold,
    fontWeight: "600",
    fontSize: 32,
    alignSelf: "center",
    marginTop: 25,
  },
  textArea: {
    borderWidth: 0,
    fontSize: 24,
    minHeight: 40,
    fontFamily: appFonts.InterRegular,
    marginBottom: 5,
    marginTop: 10
  },
  maxLength: {
    fontSize: 14,
    color: 'rgba(0,0,0,0.5)'
  },
  container: {
      paddingLeft: 16,
      paddingRight: 16
  },
  button: {
    width: widthPercentageToDP(30),
    height: 43,
    borderWidth: 1,
    borderColor: "black",
    justifyContent: "center",
    alignItems: "center",
    borderRadius: 4,
    alignSelf: "center",
    marginTop: 20,
    marginBottom: 20,
  },
  buttonText: {
    fontFamily: appFonts.InterBold,
    fontSize: 18,
  },
});

export default styles;
