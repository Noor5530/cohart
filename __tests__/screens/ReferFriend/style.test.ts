import "react-native";
import styles from "../../../app/screens/ReferFriend/styles";

test("styles", () => {
  const style = styles;

  expect(style).toMatchSnapshot();
});
