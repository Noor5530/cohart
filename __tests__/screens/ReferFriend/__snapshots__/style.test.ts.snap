// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`styles 1`] = `
Object {
  "bottomContainer": Object {
    "alignItems": "center",
    "flex": 1,
    "width": "100%",
  },
  "bottomText": Object {
    "alignItems": "center",
    "flexDirection": "row",
    "height": 40,
    "justifyContent": "center",
  },
  "bottonText": Object {
    "fontFamily": "Inter-Regular",
    "fontSize": 13,
    "fontWeight": "500",
  },
  "button": Object {
    "alignItems": "center",
    "borderColor": "#000000",
    "borderRadius": 30,
    "borderWidth": 1,
    "height": 35,
    "justifyContent": "center",
    "width": 140,
  },
  "buttonContainer": Object {
    "alignItems": "center",
    "flex": 1,
    "marginTop": undefined,
    "paddingHorizontal": undefined,
    "width": "100%",
  },
  "buttonStyles": Object {
    "alignItems": "center",
    "borderRadius": 39,
    "borderWidth": 0.5,
    "height": undefined,
    "justifyContent": "center",
    "marginVertical": undefined,
    "width": undefined,
  },
  "buttonTextStyle": Object {
    "color": "#000000",
    "fontWeight": "700",
    "textTransform": "uppercase",
  },
  "container": Object {
    "alignItems": "center",
    "backgroundColor": "#FFFFFF",
    "height": undefined,
    "justifyContent": "space-between",
    "width": "100%",
  },
  "cross": Object {
    "alignItems": "center",
    "borderRadius": 10,
    "borderWidth": 1,
    "height": 20,
    "justifyContent": "center",
    "marginRight": 10,
    "width": 20,
  },
  "header": Object {
    "width": "100%",
  },
  "label": Object {
    "color": "#000000",
    "fontSize": 13,
  },
  "mainContainer": Object {
    "alignItems": "center",
    "flex": 1,
    "paddingHorizontal": undefined,
    "width": "100%",
  },
  "phoneCodeTextStyle": Object {
    "color": "#000000",
    "fontFamily": "Inter-Regular",
    "fontSize": undefined,
    "fontWeight": "bold",
    "margin": 0,
  },
  "phoneContainerStyle": Object {
    "alignItems": "flex-start",
    "backgroundColor": "#FFFFFF",
    "borderColor": "#000000",
    "borderRadius": 14.4067,
    "height": 50.42,
    "justifyContent": "flex-start",
    "margin": 0,
    "padding": 0,
    "paddingLeft": 20,
    "width": "100%",
  },
  "phoneCountryContainerStyle": Object {
    "backgroundColor": "#FFFFFF",
    "borderColor": "#000000",
    "borderRadius": 10,
    "borderWidth": 2,
    "paddingHorizontal": 10,
  },
  "phoneTextContainerStyle": Object {
    "backgroundColor": "#FFFFFF",
    "height": 50.42,
    "margin": 0,
    "width": "90%",
  },
  "phoneTextInputTextInputStyle": Object {
    "color": "#000000",
    "fontFamily": "Inter-Regular",
    "fontSize": undefined,
    "fontWeight": "bold",
    "height": 50.42,
    "margin": 0,
    "padding": 0,
    "width": "90%",
  },
  "reefersText": Object {
    "color": "#000000",
    "fontSize": 13,
    "height": 55,
    "paddingTop": 25,
    "textAlignVertical": "top",
  },
  "summaryText": Object {
    "color": "#000000",
    "fontFamily": "Inter-Regular",
    "fontSize": 15,
    "paddingLeft": undefined,
    "paddingTop": 10,
    "width": undefined,
  },
}
`;
